org.videolan.VLC.json

```diff
@@ -0,0 +1,867 @@
+{
+  "app-id": "org.videolan.VLC",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "branch": "2.2.4",
```

Comment:
Remove the branch, it should be set to `stable` by default I believe which allows users to get updates within that same branch.